x-pack/plugins/lens/public/xy_visualization/xy_config_panel.tsx

```diff
@@ -481,12 +488,41 @@ const idPrefix = htmlIdGenerator()();
 
 export function DimensionEditor(props: VisualizationDimensionEditorProps<State>) {
   const { state, setState, layerId, accessor } = props;
+  const horizontalOnly = isHorizontalChart(state.layers);
+  const chartHasMoreThanOneSeries =
+    state.layers.length > 1 ||
+    state.layers.some(({ accessors }) => accessors.length > 1) ||
+    state.layers.some(({ splitAccessor }) => splitAccessor);
```

Comment:
I was expecting that we'd offer the "show value labels" option in a different place and in more cases than you're showing here:

* I was hoping to show this in the overall chart options
* I think we can support the value labels as long as it's a bar chart with only one series, horizontal or vertical